aerogear/digger-installer
# AGDIGGER-132 fix android pod run command

JIRA: https://issues.jboss.org/browse/AGDIGGER-132

### Verification:

1. Delete Android pod template config in Jenkins
2. Run digger installer (e.g. `ansible-playbook -i cluster-up-example sample-build-playbook.yml
 --tags=configure-buildfarm`)
3. Check Android pod template in Jenkins again. "Command to run slave agent" should be empty.
4. Build any Android application to see if Android pod is spawned correctly (no handy project? use https://github.com/aliok/android25sampleapp.git)
